### ISSUE TITLE
Jaws of Life can't open unpowered doors.

### ISSUE THREAD

**Issue by Lady-Luck** [ParadiseSS13/Paradise#6762]
During a complete station power outage, I was unable to open doors with the jaws of life. A normal crowbar worked.
